data/or/municipalities/Kent-Studebaker-40f2aa06-452b-4152-8ee2-d935e9d37495.yml

```diff
@@ -1,11 +1,11 @@
 id: ocd-person/40f2aa06-452b-4152-8ee2-d935e9d37495
 name: Kent Studebaker
-given_name: Kent
-family_name: Studebaker
+given_name: Joe
+family_name: Buck
```

Comment:
Can't tell if this was an update or a mistake? Looks like the old mayor's term might have ended but I didn't look super closely.

Reply:
I'm not 100% sure on this one. 

AFAIKT Joe Buck is the newly elected mayor of Lake Oswego, OR, but the outgoing mayor (Kent Studebaker) has `end_date: 2021-03-01` so maybe he remains in office (and is actually the active mayor) until this coming March?

Reply:
> Can't tell if this was an update or a mistake? Looks like the old mayor's term might have ended but I didn't look super closely.

...so assuming we have these two active mayor records for the same jurisdiction...does openstates resolve that automatically? Or maybe need to resolve by adding a `start_date: 2021-03-01` for Joe Buck (assuming that's really when the new term begins)?

Reply:
I went ahead and fixed that.